modules/tensor_mechanics/src/materials/ADRadialReturnStressUpdate.C

```diff
@@ -232,9 +232,10 @@ ADRadialReturnStressUpdate::maximumPermissibleValue(const ADReal & effective_tri
 Real
 ADRadialReturnStressUpdate::computeTimeStepLimit()
 {
-  Real scalar_inelastic_strain_incr = MetaPhysicL::raw_value(_effective_inelastic_strain[_qp]) -
-                                      _effective_inelastic_strain_old[_qp];
-  if (MooseUtils::absoluteFuzzyEqual(scalar_inelastic_strain_incr, 0.0))
+  const Real scalar_inelastic_strain_incr =
+      std::abs(MetaPhysicL::raw_value(_effective_inelastic_strain[_qp]) -
+               _effective_inelastic_strain_old[_qp]);
+  if (!scalar_inelastic_strain_incr)
```

Comment:
I changed this away from fuzzy eval...I don't think its necessary, and a small strain should still reproduce a timestep value that works.